backend/app/views/spree/admin/payment_methods/index.html.erb

```diff
@@ -24,10 +24,10 @@
     </colgroup>
     <thead>
       <tr data-hook="admin_payment_methods_index_headers">
-        <th><%= Spree.t(:name) %></th>
-        <th><%= Spree.t(:provider) %></th>
-        <th><%= Spree.t(:display) %></th>
-        <th><%= Spree.t(:active) %></th>
+        <th><%= Spree::PaymentMethod.human_attribute_name(:name) %></th>
+        <th><%= Spree::PaymentMethod::human_attribute_name(:type) %></th>
```

Comment:
This has `::` before the method call, where you want  a single `.` (also the two lines below).